SQL_SELECT_Mariam_Poghosyan_HW.sql

```diff
@@ -0,0 +1,133 @@
+--1-1
+SELECT f.title
+FROM public.film f
+INNER JOIN public.film_category fc ON f.film_id = fc.film_id 
+INNER JOIN public.category c ON fc.category_id = c.category_id
+WHERE c.name = 'Animation' 

```

Comment:
Let's imagine that in my version of the database, all categories are listed in the lower case. In this case, it is safest to bring all the text fields to the same style.
It would be:
lower(c.name) = 'animation' or upper(c.name) = 'ANIMATION'
This will help to avoid unexpected mistakes. I strongly recommend using one of the instructions in all text comparisons.